doc/src/manual/modules.md

```diff
@@ -116,24 +116,32 @@ end
 
 ### Standard modules
 
-There are three important standard modules: Main, Core, and Base.
+There are three important standard modules: `Main`, `Core`, and `Base`.
 
-Main is the top-level module, and Julia starts with Main set as the current module.  Variables
-defined at the prompt go in Main, and `varinfo()` lists variables in Main.
+`Main` is the top-level module, and Julia starts with `Main` set as the current module.  Variables
+defined at the prompt go in `Main`, and [`varinfo()`](@ref) lists variables in `Main`.
 
-Core contains all identifiers considered "built in" to the language, i.e. part of the core language
+`Core` contains all identifiers considered "built in" to the language, i.e. part of the core language
 and not libraries. Every module implicitly specifies `using Core`, since you can't do anything
 without those definitions.
 
-Base is the standard library (the contents of base/). All modules implicitly contain `using Base`,
+`Base` is part of the standard library (the contents of `base/`). All modules implicitly contain `using Base`,
```

Comment:
I think we should differentiate between `Base` and stdlib.

Reply:
Agreed. Any suggestions?

Reply:
... Bueller?